geth/jail/jail.go

```diff
@@ -67,6 +67,16 @@ func (jail *Jail) NewCell(chatID string) (common.JailCell, error) {
 	return cell, nil
 }
 
+// Stop stops jail and all assosiacted cells.
+func (jail *Jail) Stop() {
+	jail.cellsMx.Lock()
```

Comment:
Sometimes a mutex is called `lock`, sometimes `mx`. I know them as `mux`. At least we should try to find our common convention. Additionally it's usual to have a `defer jail.cellsMx.Unlock()` direct behind the `Lock()`.

Reply:
Right, I think we use `nameMx` convention (at least, according to previous PR), but maybe worth adding to our guidelines.
As for the defer, for such small functions unlocking via `defer` or directly is practically equal, but defer has some small performance penalty. If function grows, it's easy to switch to `defer` here.

Reply:
Do we have a kind of performance problem here? I prefer to follow conventions over premature optimization.

Reply:
@themue I totally agree here. I originally thought we're commenting single line function, but yeah it's a bit more than one line here, so defer makes more sense. Done.